netdiff/utils.py

```diff
@@ -134,35 +179,26 @@ def _netjson_networkgraph(
         raise NetJsonError('version cannot be None except when protocol is "static"')
     # prepare nodes
     node_list = []
-    for node in nodes:
-        netjson_node = OrderedDict({'id': node[0]})
+    for ip, data in nodes:
+        netjson_node = OrderedDict({'id': ip})
         # must copy properties dict to avoid modifying data
-        properties = node[1].copy()
-        local_addresses = properties.pop('local_addresses', None)
-        label = properties.pop('label', None)
-        # append local_addresses only if not empty
-        if local_addresses:
-            netjson_node['local_addresses'] = local_addresses
-        # append properties only if not empty
-        if properties:
-            netjson_node['properties'] = properties
-        # append label only if not empty
-        if label:
-            netjson_node['label'] = label
+        properties = data.copy()
+        netjson_node['label'] = properties.pop('label', '') or ''
```

Comment:
isn't `or ''` redundant?

Reply:
Not really!

`properties.pop('label', '')` will return the empty string if the label key does not exist in properties, but sometimes it exists with the None value, so None is returned. Adding `or ''` is ugly but makes sure that the default value is the empty string, rather than None

Reply:
ok I see, what about `properties.pop('label') or ''` ? Minor detail anyway.

Reply:
That will crash when the label is not present (which can happen because it's optional). I will try to find a way to make it prettier

Reply:
no worries it's fine :+1: